### PR TITLE
librsvg: Update to 2.59.1

### PR DESCRIPTION
**Pull request by lazka**
so we can move to meson, and don't have to deal with libtool at least

### CODE REVIEW

File: mingw-w64-librsvg/0002-avoid-link-whole.patch [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
++    link_with: librsvg_c_lib,
+     link_args: link_args,
+     link_depends: librsvg_ver,
+     dependencies: library_dependencies,
+@@ -342,7 +342,7 @@
+       librsvg_gir = shared_library(
+         'rsvg-gir',
+         rsvg_dummy,
+-        link_whole: librsvg_c_lib,
++        link_whith: librsvg_c_lib,
```
Comment (ognevny) [inline mingw-w64-librsvg/0002-avoid-link-whole.patch:17]: typo

Comment (lazka) [reply mingw-w64-librsvg/0002-avoid-link-whole.patch:17]: oops, thanks

**Comment by ognevny** [comment]
also PKGBUILD L17: change autotools to meson :)

**Comment by lazka** [comment]
I'll try to upstream the gdk-pixbuf patch once this builds.

**Comment by lazka** [comment]
Two issues:

* cargo-c uses the wrong linker for the static build, worked around by not linking anything
* binutils strip fails to strip the static archive, which is now huge

**Comment by lazka** [comment]
https://gitlab.gnome.org/GNOME/librsvg/-/merge_requests/1015

**Comment by ognevny** [comment]
patch for clang is not pushed

**Comment by lazka** [comment]
> binutils strip fails to strip the static archive, which is now huge

This is still not solved, but having a newer version is more important, so I'm merging this.

I'll try to upstream the clang patch later.

**Comment by lazka** [comment]
> I'll try to upstream the clang patch later.

https://gitlab.gnome.org/GNOME/librsvg/-/merge_requests/1052

**Pull request merged**